ui/src/app/cohort-search/list-search/list-search.component.tsx

```diff
@@ -51,6 +51,11 @@ const styles = reactStyles({
     opacity: 0.4,
     cursor: 'not-allowed'
   },
+  brandIcon: {
+    marginRight: '0.4rem',
+    color: '#9a9a9a',
```

Comment:
Please use the colors file if possible. Lou should be able to help with what the right color is. (And we're avoiding adding new colors to that file, if possible it should be a different intensity version of one of the colors in our palette)

Reply:
Okay. I'm going to do a follow-up PR to try to refactor all CB colors to use the new file.

Reply:
Sounds good. I've been going through as well, but if you can handle the CB, that would be a great help